examples/API_v2/get_retweeters.py

```diff
@@ -0,0 +1,19 @@
+import tweepy
+
+
+# Replace bearer token value with your own
+bearer_token = ""
+
+# Initializing the Tweepy client
+client = tweepy.Client(bearer_token)
+
+# Replace Tweet ID
+id = '1441054496931541004'
+
+# By default the user ID, name and username are returned. user_fields can be used 
+# to specify the additional user data that you want returned for each user e.g. profile_image_url
+users = client.get_retweeters(id=id, user_fields=['profile_image_url'])
+
+for user in users.data:
+    print(user)
```

Comment:
This will print the User object cast to a string, which will just be username of each user.
It's not really clear that that's what this does, without any explanation.